src/scoring.rs

```diff
@@ -3,8 +3,10 @@ use crate::matching::Match;
 use std::collections::HashMap;
 use std::{cmp, fmt::Display};
 
+/// Score generated when measuring the entropy of a password.
 #[derive(Debug, Clone, Copy, Hash, PartialEq, Eq, PartialOrd, Ord)]
 #[non_exhaustive]
+#[cfg_attr(feature = "ser", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
Can you verify, does this serialize as the numeric values, or as the words?

Reply:
Good catch @shssoichiro , I have now ensured it serializes and deserializes to `u8` and added a small test spec to verify.